src/FSharp.Control.TaskSeq.SmokeTests/TestUtils.fs

```diff
@@ -0,0 +1,131 @@
+namespace TaskSeq.Tests
+
+open System
+open System.Threading
+open System.Threading.Tasks
+open System.Diagnostics
+open System.Collections.Generic
+
+open Xunit
+open Xunit.Abstractions
+open FsUnit.Xunit
+
+open FSharp.Control
+
+/// Milliseconds
+[<Measure>]
+type ms
+
+/// Microseconds
+[<Measure>]
+type µs
+
+/// Helpers for short waits, as Task.Delay has about 15ms precision.
+/// Inspired by IoT code: https://github.com/dotnet/iot/pull/235/files
+module DelayHelper =
+
+    let private rnd = Random()
```

Comment:
pretty sure Random is not threadsafe (get stuck repeating the same value) under concurrent calls, though it's rare. I believe they have made it threadsafe in v7, but perhaps that's only their static one
(I dont think this is a big deal given you're going to switch to that in due course anyway)

Reply:
Thanks, I wasn't aware of that. 

The move to .NET 7.0 for SmokeTests allows access to a few more BCL methods that return an `IAsyncEnumerable<_>` (like https://learn.microsoft.com/en-us/dotnet/api/system.io.file.readlinesasync?view=net-7.0), which will improve our testing coverage.